src/commands/sbom/renderer.ts

```diff
@@ -74,3 +77,15 @@ export const renderSuccessfulCommand = (duration: number) => {
 
   return fullStr
 }
+
+export const renderPayloadWarning = (dependencies: Dependency[]): string => {
+  let ret = ''
+
+  for (const dep of dependencies) {
+    if (!validateDependencyName(dep)) {
+      ret += `invalid dependency name ${dep.name}\n`
+    }
+  }
+
+  return ret
+}
```

Comment:
Could we prettify this information similar to what we do in `renderMissingTags`, so it's easier to notice for end-users?

```suggestion
export const renderPayloadWarning = (dependencies: Dependency[]): string => {
  const errors = []

  for (const dep of dependencies) {
    if (!validateDependencyName(dep)) {
      errors.push(chalk.red(` - invalid dependency name ${dep.name}\n`))
    }
  }

  if (errors.length === 0) {
    return ''
  }

  let baseHeading = 'The following issues were detected:\n'

  return baseHeading += errors.join('')
}
```